src/hooks/useUnlinkContact.ts

```diff
@@ -19,10 +19,10 @@ const useUnlinkContact: UseUnlinkContact = () => {
   const { context } = useDeskproLatestAppContext() as { context: UserContext };
   const { asyncErrorHandler } = useAsyncError();
   const [isLoading, setIsLoading] = useState<boolean>(false);
-  const dpUserId = useMemo(() => get(context, ["data", "user", "id"]), [context]);
+  const dpUserId = context?.data?.user.id;
```

Comment:
`context` is always an object so no need for `?.`. Or if it isn't always an object, the typecast on L19 needs updating to `{ context: UserContext } | undefined`

Reply:
I'm afraid, not always. Here story for fix in `app-sdk`
https://app.shortcut.com/deskpro/story/158090/app-sdk-should-fix-context-typings

Reply:
fixed with this `const { context } = useDeskproLatestAppContext() as { context?: UserContext };` because there might not be any context at all.